contents/docs/integrate/feature-flags-code/_snippets/feature-flags-code-python.mdx

```diff
@@ -58,6 +58,20 @@ posthog.capture(
 )
 ```
 
+
+### Sending `$feature_flag_called` events
+
+By default, we send a `$feature_flag_called` event whenever you call `posthog.get_feature_flag()` or `posthog.is_feature_enabled()` and it's a new user, or the value of the flag has changed. This is useful for analytics, and knowing when a flag was accessed for a user.
+
+> *Note:* Tracking whether it's a new user, or whether the flag value is changed happens in a local cache, which means if you reinitialise the PostHog client, this cache will reset as well, and can end up sending events again for old users as well. This is not a problem. 
+
+You can disable this by setting the `send_feature_flag_events` in your function call, like so:
```

Comment:
Why would you want to disable this? Example scenario?

Reply:
For example, when this would add too many unnecessary events, or you don't need the analytics, or it's being called at such a high volume that sending events slows things down.